### DISCUSSION TITLE
Issue slicing certain models

### DISCUSSION THREAD

**Discussion by Edzi11** [OrcaSlicer/OrcaSlicer#1299] [Q&A]
As stated I'm having some issues slicing certain models when using supports. It freezes after getting to 65%. The only time it doesn't
 is when I'm using slim tree supports or default normal supports. Oddly enough cura has no issue with the models and slices it within minutes. Attached is the model that isnt working correctly. File is about 1.8mb and about 40k triangles but I have sliced bigger files in cura without issue. any suggestions as I want this to be my main slicer but not if it cant slice every file i throw at it like cura does

![model](https://github.com/SoftFever/OrcaSlicer/assets/123626106/09fb7b4d-71a0-49f8-8c9b-bb3791101655)

**Comment by hliebscher** [comment]
for me its fine 
![tree](https://github.com/SoftFever/OrcaSlicer/assets/9274162/a5092da8-cd34-4c62-b795-9a264925c99a)
